From: Josh Law <objecting@objecting.org>
Subject: MAINTAINERS: add Josh Law as reviewer for library code
Date: Sat, 7 Mar 2026 22:21:54 +0000

Add myself as a designated reviewer for the library code to help review
incoming patches and improvements.

Link: https://lkml.kernel.org/r/20260307222154.2848660-1-objecting@objecting.org
Signed-off-by: Josh Law <objecting@objecting.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    1 +
 1 file changed, 1 insertion(+)

--- a/MAINTAINERS~maintainers-add-josh-law-as-reviewer-for-library-code
+++ a/MAINTAINERS
@@ -14611,6 +14611,7 @@ F:	tools/testing/nvdimm/
 
 LIBRARY CODE
 M:	Andrew Morton <akpm@linux-foundation.org>
+R:	Josh Law <objecting@objecting.org>
 L:	linux-kernel@vger.kernel.org
 S:	Supported
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/akpm/mm.git mm-nonmm-unstable
_
